specifyweb/frontend/js_src/lib/components/Merging/InvalidMergeRecords.tsx

```diff
@@ -13,32 +13,40 @@ import { commonText } from '../../localization/common';
 import { mergingText } from '../../localization/merging';
 import { Tables } from '../DataModel/types';
 import { TableIcon } from '../Molecules/TableIcon';
+import { className } from '../Atoms/className';
 
 function InvalidMergeRecords({
   resources,
   tableName,
-  body = '',
-  specificText,
+  specificText = '',
 }: {
   readonly tableName: keyof Tables;
   readonly resources: RA<SerializedResource<AnySchema>>;
-  readonly body?: string;
   readonly specificText?: string;
 }): JSX.Element {
   return (
     <div className="flex flex-1 flex-col gap-4">
-      <H2>{body}</H2>
+      <H3 className={className.headerPrimary}>
```

Comment:
I think you meant to do `<h3 className={className.headerPrimary}>` not `<H3 className={className.headerPrimary}>`, right? (note the lowercase `h3`)

`H3` comes with it's own styles, and then you overwrite those by adding `className={className.headerPrimary}`, so it's kind of redundant

unless you intentionally meant to merge the two styles together, but why?